lago/sdk.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+from lago import cmd
+from sdk_utils import SDKWrapper
+
+
+def init(config, workdir=None, **kwargs):
+    """
+    Initialize the Lago environment
+
+    Args:
+        config(str): Path to LagoInitFile
+        workdir(str): Path to initalize the workdir
+        **kwargs(dict): Pass arguments to :func:`~lago.cmd.do_init`
+
+    Returns:
+        :class:`~lago.sdk.sdk`: Initialized Lago enviornment
+
+    Raises:
+       :exc:`~lago.utils.LagoException`: If initialization failed
+    """
+
+    # .. to-do:: load global configuration from a file
+    # .. to-do:: allow loading the env from an existing directory
+    defaults = {
+        'workdir': workdir,
+        'template_repo_path': 'http://templates.ovirt.org/repo/repo.metadata',
+        'template_store': '/var/lib/lago/store',
+        'template_repos': '/var/lib/lago/repos',
+        'set_current': True,
+        'skip_bootstrap': False,
+        'skip_build': False,
+        'prefix_name': 'default',
+    }
+    defaults.update(kwargs)
+    defaults['virt_config'] = config
+    workdir, prefix = cmd.do_init(**defaults)
```

Comment:
I Agree, I think that will involve much more re factoring that I think is wanted for this patch. I wanted to provide the minimum framework needed in order to start using the SDK, and in parallel let us refactor it without breaking existing features.